apis/core-api/src/core/services/AwsS3Service.ts

```diff
@@ -0,0 +1,50 @@
+import { v4 as uuidv4 } from 'uuid';
+import awsSdk from 'aws-sdk';
+import { buildFileName } from '@ultras/utils';
+import { awsConfig } from 'config';
+import BaseService from './BaseService';
+
+const bucketFolders: Record<string, string> = {
+  profilePicture: 'profile-pictures',
```

Comment:
It would be better to keep this service pure,,and inject supported folder types from the usage

Reply:
okay, I'll move it in constants.ts